### PR TITLE
[W5][T09-2]Li Zeyang

### PR DESCRIPTION
**Pull request by ghost**
Week 5 2KLoC

### CODE REVIEW

**Comment by BrittonAlone** [comment]
It is a very funny feature, showing respect to the users. And it is related to the reality, since it's still CNY.

**Comment by Wanchunwei** [comment]
It's a very kind modification, hhh!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -91,6 +91,18 @@ Returns `John Doe` but not `john`.
 * `find Betsy Tim John` +
 Returns Any person having names `Betsy`, `Tim`, or `John`.
 
+== Happy Chinese New Year: "HappyCNY"
+
+This application will represent the coders to say Happy Chinese New Year to the users
```
Comment (RubaP) [inline docs/UserGuide.adoc:96]: Good that you have updated the UserGuide.

File: src/seedu/addressbook/commands/HappyCNY.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ */
+public class HappyCNY extends Command {
+
+    public static final String COMMAND_WORD = "HappyCNY";
+    public static final String MESSAGE_USAGE = "It is not CNY.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Address book is greeting you!";
+
+    public HappyCNY() {
```
Comment (RubaP) [inline src/seedu/addressbook/commands/HappyCNY.java:14]: If the constructor does not perform any meaningful functionality, you can remove it.

**Review by RubaP — COMMENTED** [review]
Good attempt. But try to work on more meaningful functional enhancement in future so that it will help you to learn more SE practice through thinking and coding. Moreover, always add test cases as a practice whenever you make a functional change to avoid regression errors in the future

**Pull request closed without merging**